crates/bindings-csharp/Runtime/Runtime.cs

```diff
@@ -77,6 +59,14 @@ public AlgebraicType GetAlgebraicType(ITypeRegistrar registrar) =>
     }
 }
 
+public abstract class BaseReducerContext<DbView> : DbContext<DbView>, IReducerContext
+    where DbView : class, new()
```

Comment:
Unless we have an interface to uphold, I'd rather not spell out extra constraints. In this case it doesn't seem to add much value in terms of checks.

```suggestion
    where DbView : new()
```

Reply:
I made it a class because it does carry state; the connection handle in this case. A struct could be copied/defaulted and introduce a silent null connection. This forces it to be a reference type, and given there's only one per application the distinction between value/ref is no longer relevant, I think.

Reply:
Well in this case Local is stateless, but the same base class has to support DbConnection on the client side.